docs/guide/shop_features/shop_features.md

```diff
@@ -31,7 +31,7 @@ They can also upload a CSV file containing a list of SKUs and quantities.
 
 ## Order history [[% include 'snippets/commerce_badge.md' %]]
 
-When an ERP system is connected, the customer can browse and search past orders, invoices, delivery notes, and credit memos.
+When an ERP system is connected, the customer can browse and search [past orders, invoices, delivery notes, and credit memos](../order_history/order_history.md).
```

Comment:
```suggestion
When an ERP system is connected, the customer can browse and search [past orders, invoices, delivery notes, and credit memos](../order_history/orderhistory.md).
```

Also - the title of the target article lacks a space (to be fixed in the mkdocs.yaml file)

Reply:
Forgot to mention - the file name will be changed in https://github.com/ezsystems/developer-documentation/pull/1331